config/config.go

```diff
@@ -94,7 +97,7 @@ type BaseConfig struct {
 	FilterPeers bool `mapstructure:"filter_peers"` // false
 
 	// What indexer to use for transactions
-	TxIndex string `mapstructure:"tx_index"`
+	TxIndexer string `mapstructure:"tx_index"`
```

Comment:
should this move into the new `TxIndex` config ?

Reply:
yes